dart/lib/src/client_reports/outcome.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:meta/meta.dart';
+
+@internal
+enum Outcome { ratelimitBackoff, networkError }
+
+extension OutcomeExtension on Outcome {
+  String toStringValue() {
+    switch (this) {
```

Comment:
I guess you are aware that we have more reasons like sample_rate, before_send, see https://develop.sentry.dev/sdk/client-reports/#envelope-item-payload, and you are going to add these in a future PR.

Reply:
Yes, just wanted to have a starting point, will add the others too.